### PR TITLE
Workspace support

### LINKED ISSUE #4: Support for workspaces
Consider adding support to use cargo kcov in a [workspace](http://doc.crates.io/manifest.html#the-workspace-section) project.

You can see with `cargo +nightly build --help` the kind of options for workspace projects.

### PR DESCRIPTION
**Pull request by bjgill**
This adds support for workspaces by taking advantage of `cargo test --all`.

Note that this will break support for Rust 1.14.0 (where cargo test does not have the `--all` flag).

Fixes #4

### CODE REVIEW

**Comment by bjgill** [comment]
I don't know how strongly you feel about retaining support for Rust 1.14.0. A possible alternative would be to have this as a Rust 1.16.0-only (+above) feature.

**Comment by bjgill** [comment]
Will re-submit once tests are fixed.

**Comment by bjgill** [comment]
OK - having two 'lib.rs' in /specimen was causing problems.

**Comment by bjgill** [comment]
CI is currently failing due to #8 (but now nothing else, as far as I can tell).

**Comment by kennytm** [comment]
Thanks for the PR! I'll merge once I figure out what is going on in #8.

File: src/main.rs [hunk 1]
```diff
@@ -126,25 +127,29 @@ fn run(matches: &ArgMatches) -> Result<(), Error> {
...
-        if is_verbose {
-            write_msg("Build", "test executables");
-        }
-        try!(build_test(matches))
-    };
+            if is_verbose {
+                write_msg("Build", "test executables");
+            }
+            try!(build_test(matches))
+        };
```
Comment (kennytm) [inline src/main.rs]: Why everything is re-indented here? rustfmt?

Comment (bjgill) [inline src/main.rs]: Most likely - I'll revert the formatting for consistency

**Pull request merged**